h2o-core/src/main/java/water/AutoBuffer.java

```diff
@@ -1004,17 +1006,28 @@ AutoBuffer putUdp(UDP.udp type, int senderPort){
    *
    * @param type type of the UDP datagram
    */
-  AutoBuffer putUdp (UDP.udp type) {
-    return putUdp(type, H2O.H2O_PORT); // Outgoing port is always the sender's (me) port
+  AutoBuffer putUdp(UDP.udp type) {
```

Comment:
@jakubhava is there is a way for sparkling water client to use TCP instead of UDP? 
I think it can be harder to make UDP communication stable. 
Thanks.

Reply:
It's just confusing name, we always use TCP for all the communication. Sorry for confusion

Reply:
Good to know . Thanks @jakubhava